lstchain/calib/camera/calibrator.py

```diff
@@ -119,11 +119,16 @@ def _initialize_correction(self):
                     # read the calibration data for the moment only one event
                     table = '/tel_' + str(telid) + '/calibration'
                     next(h5_table.read(table, self.mon_data.tel[telid].calibration))
-                    # eliminate inf values (should be done probably before)
-                    dc_to_pe = self.mon_data.tel[telid].calibration.dc_to_pe
 
+                    dc_to_pe=self.mon_data.tel[telid].calibration.dc_to_pe
+                    # put to zero unusable pixels
+                    dc_to_pe[self.mon_data.tel[telid].calibration.unusable_pixels] = 0
+                    # eliminate inf values id any (should be done probably before)
                     dc_to_pe[np.isinf(dc_to_pe)] = 0
-                    self.log.info(f"read {self.mon_data.tel[telid].calibration.dc_to_pe}")
+
+                    # read the pixel_status container
+                    table = '/tel_' + str(telid) + '/pixel_status'
```

Comment:
Sorry @FrancaCassol, where is the `pixel_status` being filled and with which values?

Reply:
Hi @rlopezcoto,
The pixel status container is filled by the calibration script on the base of the inputs from hardware (R0), pedestals and flat fields